app/routes.js

```diff
@@ -14,5 +14,6 @@ exports.init = (app) => {
   //Album endpoint routes
   app.get('/albums', auth.checkCredentials, album.list);
   app.post('/albums', [auth.checkCredentials, albumValidator.validAlbumId], album.purchase);
+  app.get('/users/albums/photos', auth.checkCredentials, album.photoList);
```

Comment:
As we talked in slack the route should be something like this `/users/albums/:albumId/photos`